net-libs/libslirp/libslirp-4.7.0.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A TCP-IP emulator used to provide virtual networking services"
+DESCRIPTION="libslirp is a general purpose TCP-IP emulator used by virtual machine hypervisors, \
+containers, and emulation software to provide virtual networking services."
+HOMEPAGE="https://gitlab.freedesktop.org/slirp/libslirp/"
+COPYRIGHT="1995, 1996 Danny Gasparovski"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE-/archive/v$portVersion/libslirp-v$portVersion.tar.gz"
+CHECKSUM_SHA256="9398f0ec5a581d4e1cd6856b88ae83927e458d643788c3391a39e61b75db3d3b"
+SOURCE_DIR="libslirp-v$portVersion"
+PATCHES="libslirp-$portVersion.patchset"
+
+ARCHITECTURES="all ?x86_gcc2"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
+
+PROVIDES="
+	libslirp$secondaryArchSuffix = $portVersion
+	lib:libslirp$secondaryArchSuffix = 4
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libslirp${secondaryArchSuffix}_devel = $portVersion
+	devel:libslirp$secondaryArchSuffix = 4
+	"
+REQUIRES_devel="
+	libslirp$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libglib_2.0$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+    cmd:cmake
```

Comment:
Is cmake actually being used?

Reply:
No, was left in there from the recipe I adapted, I've removed it now :)

Reply:
Haikuporter comes with a few generic recipes, if you add them to the templates you could use them when starting on a new recipe (I do it also, I even created a generic python recipe on the other laptop for this.
![generic-recipes](https://user-images.githubusercontent.com/16057090/167074207-2dcc04e8-e096-42f2-b7c1-65991ff00740.png)